ts/utils/wallet.ts

```diff
@@ -0,0 +1,22 @@
+import { Wallet } from "../types/pagopa";
+
+/* 
+    Contains utility functions to check conditions
+    used across project (currently just in CardComponent)
+ */
+
+export const isExpiredCard = (w: Wallet) => {
+  const today: Date = new Date();
+  const cmpM: number = today.getMonth() + 1;
+  const cmpY: number = parseInt(
+    today
+      .getFullYear()
+      .toString()
+      .slice(2),
+    10
+  );
+  return (
+    parseInt(w.creditCard.expireMonth, 10) > cmpM &&
+    parseInt(w.creditCard.expireYear, 10) <= cmpY
+  );
+};
```

Comment:
@linsock 
If this function means: "return true when a cc is expired" then the check produces wrong results for certain inputs: i.e 03/19 (return false) or 06/19 (return true)
It should be something:
```javascript 
return (parseInt(w.expireYear, 10) < cmpY ||
parseInt(w.expireYear, 10) == cmpY && parseInt(w.expireMonth, 10) < cmpM));
```
you should write some tests on this check